### PR TITLE
Add file to Playbook/ Business

### PR DESCRIPTION
**Pull request by Lizzie14**
Add 5 files:
1. Business playbook outline
2. Collaboration guideline
3. DF Workflow
4. General questions for service feedback
5. How to work with Clients - Account View

#### What's this PR do?

- [x] Add new routes for user
- [x] Add instruction for using docker-compose

#### What are the relevant Git tickets?

// Put in link to Git Issue

#### Screenshots (if appropriate)

// Use [Licecap](http://www.cockos.com/licecap/) to share a screencast gif.

#### Any background context you want to provide? (if appropriate)

- Is there a blog post?
- Does the knowledge base need an update?
- Does this add new dependencies which need to be added to?

### CODE REVIEW

File: business/Collaboration Guideline.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Collaboration Guideline
```
Comment (huynguyenh) [inline business/Collaboration Guideline.md:1]: rename the file name to collaboration-guideline.md

File: business/DF workflow.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+* Monday:
```
Comment (huynguyenh) [inline business/DF workflow.md:1]: rename the file name to df-workflow

File: business/General question for service feedbacks.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# GENERAL QUESTIONS FOR SERVICE FEEDBACKS
```
Comment (huynguyenh) [inline business/General question for service feedbacks.md:1]: rename the filename

File: business/How To Work With Clients - Account view.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# TRAINING COURSE: HOW TO WORK WITH CLIENT
```
Comment (huynguyenh) [inline business/How To Work With Clients - Account view.md:1]: rename the filename

**Review by huynguyenh — CHANGES_REQUESTED** [review]

**Review by huynguyenh — APPROVED** [review]

**Comment by tieubao** [comment]
@huynguyenh please double check if it still has any misleading information

**Pull request merged**